xbmc/network/websocket/WebSocket.cpp

```diff
@@ -242,7 +242,7 @@ CWebSocketFrame::~CWebSocketFrame()
 
   if (m_free && m_data != NULL)
   {
-    delete m_data;
+    delete[] m_data;
```

Comment:
@Paxxi what do you think? do you see a better solution without the need of touching too much existing code?